packages/prover/tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.build.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "outDir": "lib",
+    // Had to add it because compiling error `Duplicate identifier 'Buffer'`
```

Comment:
i think the commend here will break json

Reply:
Github parse JSON as normal json files where comments are not allowed. And TS parse these as JSON5 files where comments are allowed.